cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServlet.java

```diff
@@ -1034,13 +1034,15 @@ private boolean getState(HttpServletRequest request, HttpServletResponse respons
           Map<TopicPartition, List<SampleExtrapolation>> sampleFlaws = state.monitorState().sampleExtrapolations();
           if (sampleFlaws != null && !sampleFlaws.isEmpty()) {
             for (Map.Entry<TopicPartition, List<SampleExtrapolation>> entry : sampleFlaws.entrySet()) {
-              out.write(String.format("%n%s: %s", entry.getKey(), entry.getValue()).getBytes(StandardCharsets.UTF_8));
+              out.write(String.format("%n%s: %d", entry.getKey(), entry.getValue()).getBytes(StandardCharsets.UTF_8));
```

Comment:
Not sure why do we need to change to %d here. I think the `entry.getValue()` is `List<SampleExtrapolation>`. Wondering if I missed something here.